samples/bluetooth/peripheral_bms/README.rst

```diff
@@ -0,0 +1,102 @@
+.. _peripheral_bms:
+
+Bluetooth: Peripheral Bond Management Service (BMS)
+###################################################
+
+The peripheral BMS sample demonstrates how to use the :ref:`bms_readme`.
+
+Overview
+********
+
+When connected, the sample waits for Client requests to perform any bond deleting operation.
+
+It supports up to two simultaneous Client connections.
+
+Requirements
+************
+
+* One of the Nordic development boards:
+
+  * |nRF5340DK|,
+  * |nRF52840DK|,
+  * |nRF52DK|.
+
+* A Bluetooth Low Energy dongle or development kit.
+
+* nRF Connect for Desktop.
+
+User interface
+**************
+
+LED 1:
+   * It blinks with a period of 2 seconds with the duty cycle set to 50% when the main loop is running, and the device is advertising.
```

Comment:
```suggestion
   * Blinks with a period of 2 seconds with the duty cycle set to 50% when the main loop is running and the device is advertising.
```
Also , do not put an asterisk here in these User interface descriptions. These are not supposed to be lists because almost never will there be more than one item here. Follow the format from the template:
https://github.com/nrfconnect/sdk-nrf/pull/2632/files (around line 95)

Reply:
fixed